src/js/controller.js

```diff
@@ -0,0 +1,70 @@
+import { Address } from './single-address';
+import { setInitialFormValues } from './helpers';
+
+export class Controller {
+
+  constructor(store, view) {
+    this.store = store;
+    this.view = view;
+    
+    this.initialFormState = setInitialFormValues();
+
+    this.addressList = this.store.fetchAddressList();
+
+    this.view.bindAddAddress(this.addNewAddress.bind(this));
+    this.view.bindRemoveAddress(this.removeAddress.bind(this));
+    this.view.bindEditAddress(this.editAddress.bind(this));
+    this.view.bindSaveEditedAddress(this.saveEditedAddress.bind(this));
+    this.view.bindCancelEditAddress(this.cancelEditAddress.bind(this));
+  };
+
+  addNewAddress() {
+    const addressConfig = this.view.getFormInputsValues();
+    const list = this.store.fetchAddressList();
```

Comment:
Takie coś powinno mieć miejsce w modelu. Powinieneś po prostu zrobić `this.store.addNewAddress(address)` czy coś w tym stylu